pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -2126,6 +2128,12 @@ double Object_getDoubleValue(JSOBJ Py_UNUSED(obj), JSONTypeContext *tc) {
     return GET_TC(tc)->doubleValue;
 }
 
+const char *Object_getBigNumStringValue(JSOBJ obj, JSONTypeContext *tc) {
+    // Question (arw2019): is this function right? 
+    // Do I need the outLen argument for example?
```

Comment:
Yea you need to modify the outLen pointer to let the serializer know how many bytes it should write out